lib/map.jsx

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+var MapComponent = React.createClass({
+
+    propTypes: {
+      accessToken: React.PropTypes.string.isRequired,
+      mapId: React.PropTypes.string.isRequired,
```

Comment:
I think `mapboxId` might be a better name for this one, to disambiguate it from the actual `id` of the `<div>`?